code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react';
+import { Link, useParams } from 'react-router-dom';
+
+import { API_KEY } from '../urls';
+
+const MovieDetails = () => {
+    const [MovieDetails, setMovieDetails] = useState({});
+    const { movieId } = useParams();
+
+    useEffect(() => {
+        const DETAILS_URL = `https://api.themoviedb.org/3/movie/${movieId}?api_key=${API_KEY}&language=en-US`;
+        
+        fetch(DETAILS_URL)
+            .then(response => response.json())
+            .then(json => setMovieDetails(json));
+    }, [movieId])
+
+    return (
+        <section className="movie-details" style={{ backgroundImage: `linear-gradient(rgba(0, 0, 0, 0) 70%, rgb(0, 0, 0) 100%) ,url(https://image.tmdb.org/t/p/w1280${MovieDetails.backdrop_path})`}}>
```

Comment:
Here you're adding the background-image to the section movie-details, so to specify how the image should be presented (`cover` instead of `repeat`), add it as styling to this class.